azerothcore/azerothcore-wotlk
# fix(DB/loot): Removes Scarlet Set items from rand_loot_temp tables

## Changes Proposed:
- Remove Scarlet Gauntlets (ID 10331) from reference_loot_templates 24056 and 526790.
- Remove Scarlet Wristguards (ID 10333) from reference_loot_template 24054.

Note that both items continue to be dropped by mobs within SM, as those drops are handled directly and not by loot templates.

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/725
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6101

## SOURCE:

links here show that the only mobs that should drop these items are in SM:
https://classic.wowhead.com/item=10331/scarlet-gauntlets
https://classic.wowhead.com/item=10333/scarlet-wristguards

## Tests Performed:

- Ran SQL, checked in Keira, verified items were now gone from referenced RLTs.
- Verified items still existed in normal drop tables for SM mobs.

## How to Test the Changes:

```
SELECT COUNT(*)
FROM `creature_template` ct 
JOIN `creature_loot_template` clt ON ct.lootid = clt.entry
JOIN `reference_loot_template` rlt ON rlt.entry = clt.Item
JOIN `item_template` it ON it.entry = rlt.Item
WHERE it.name = "Scarlet Gauntlets";
```
This should now return 0 rows. Run again with Wristguards in place of Gauntlets.

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
